x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/saml/SamlAuthenticateResponse.java

```diff
@@ -64,6 +70,8 @@ public TimeValue getExpiresIn() {
         return expiresIn;
     }
 
+    public Authentication getAuthentication() { return authentication; }
+
     @Override
     public void writeTo(StreamOutput out) throws IOException {
         out.writeString(principal);
```

Comment:
Need add the new `authentication` field here.

Reply:
Done